### ISSUE TITLE
WGAN loss evaluation

### ISSUE THREAD

**Issue by lennert130994** [eriklindernoren/Keras-GAN#175]
I took the code from the WGAN as @eriklindernoren provided it.
Now i am trying to evaluate the loss of the critic.
I plotted the loss of the critic on the real samples:

> plotted_loss = -self.critic.train_on_batch(imgs, valid)[0]

This loss is the blue line in the graph below.

![image](https://user-images.githubusercontent.com/23079356/62609323-d7509880-b901-11e9-9c4a-96cd61434882.png)

Now my question, why is it going below 0? what does this mean?

**Comment by fengwang** [comment]
Looks like a data leakage?

**Comment by lennert130994** [comment]
> Looks like a data leakage?

@fengwang can you elaborate?
